src/Issuer/IssuerBuilder.php

```diff
@@ -46,7 +46,7 @@ public function build(string $resource): IssuerInterface
         $metadata = IssuerMetadata::fromArray($metadataBuilder->build()->fetch($resource));
 
         $jwksProviderBuilder = $this->buildJwksProviderBuilder();
-        $jwksProviderBuilder->setJwksUri($metadata->getJwksUri());
+        $jwksProviderBuilder->withJwksUri($metadata->getJwksUri());
```

Comment:
I think this is a fluent method call so potentially this will want to be

```suggestion
        $jwksProviderBuilder = ($this->buildJwksProviderBuilder())->withJwksUri($metadata->getJwksUri());
```